src/lib/services/group-service.ts

```diff
@@ -302,7 +303,7 @@ export class GroupService {
             await this.groupStore.addUserToGroups(
                 userId,
                 newGroups.map((g) => g.id),
-                auditUser.username,
+                SSO_SYNC_USER,
```

Comment:
This change fixes the e2e tests. This is needed because the assumption the method `getOldGroupsForExternalUser` does is that all group sync is stored under this username.